config/initializers/secure_headers.rb

```diff
@@ -13,6 +13,7 @@
     default_src: ["'self'"],
     child_src: ["'self'"], # CSP 2.0 only; replaces frame_src
     # frame_ancestors: %w('self'), # CSP 2.0 only; overriden by x_frame_options in some browsers
+    form_action: ["'self'"],
```

Comment:
**question:** just to confirm, but this will turn on `form-action: 'self'` everywhere other than where `apply_secure_headers_override` is called, right?